src/App.css

```diff
@@ -1523,6 +1640,196 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
   }
 }
 
+
+/* Phones (Portrait) */
+@media (max-width: 500px) {
+  :root {
+    --mobile-orbit-size: 360px;
+    --mobile-view-width: 90vw;
+    --mobile-view-splash-body-width: 60vw;
+    --mobile-orbit-top-offset: 90px;
+  }
+
+  .orbit_closeup_back_button_mobile {
+    visibility: visible;
+    position: relative;
+    top: -35.5vh;
+    left: 86vw;
+  }
+
+  .hard_scale_value {
+    transform: scale(1) !important;
```

Comment:
What was the reason for this?  I'm assuming it has to override an inline style, but just explain so I know?

Reply:
I wrote up an explanation in the PR that actually initiated this which was the ticket where the orbits and earth size get all out of whack at a certain point. 

The problem was that the transform scale calculation for the earths orbit container hits a critical point where things don't line up anymore. 

Let me find the branch where I discuss this further.

Reply:
Here is the PR where I added that
https://github.com/rraaschfilament/across-karman/pull/16